content/v0.0.1/index.md

```diff
@@ -6,3 +6,290 @@ aliases: ["/en/"]
 # Conventional Logs v0.0.1
 
 ## Summary
+
+The conventional logs specification is an opinionated convention
+aiming to set an industry standard to ensure search-ability
+and enhance security in logs.
+
+Each log entry could be encoded in different formats, `json`, `xml`, or even a custom one.
+Most of the examples in this page will be written as `json`.
+
+A basic log entry should be structured as follows:
+
+---
+
+```
+{
+  "severity": "DEBUG",
+  "scope": "web-app",
+  "message": "could not save the user",
+  "timestamp": "2022-05-14T14:16:15+00:00"
+}
+```
+
+---
+
+A minimal log entry is composed by the following fields,
+and needs to be descriptive enough to determine the context related to event occurrence.
+
+1. The `severity`, also referred as `level`, describes the importance of a log entry.
+   Starting from the lower priority to the higher one, the recommended levels are ***debug***, ***info***,
+   ***warning*** (or ***warn***), ***error***, ***fatal***.
+3. Other values as `severity` can be used, but it is recommended to stick to the suggested one.
+4. The `scope` field represents the issuer of the log entry.
+   This is useful especially in contexts where a log entry may be created from multiple services.
+5. In complex systems the `scope` could also point to the node/service that caused the log entry.
+6. The `message` is the core information of a log entry, and describes an event that occurred in a system.
+7. When the same event happens multiple times, the `message` must always be the same and other fields must be used as
+   discriminant.
+8. The `timestamp` field determines when a log entry has been created and contains date+time, it is optional to pass
+   timezone information as well.
+10. Each of fields can be renamed and adapted to a company need. An example: `message` may be reported as `event`
```

Comment:
Nope. If a company needs to adapt an existing log solution, they should use an adapter instead of modifying the standard. A mutable standard = no standard.